redis_broker.go

```diff
@@ -224,7 +224,7 @@ func (t *RedisBroker) claim(ctx context.Context, consumer *ConsumerHandler) erro
 	return t.pool.Submit(func() {
 
 		streamKey := MakeStreamKey(Config.Redis.Prefix, consumer.Channel, consumer.Topic)
-		xAutoClaim := redisx.NewAutoClaimArgs(streamKey, consumer.Channel, 50*time.Second, "0-0", 100, consumer.Topic)
+		xAutoClaim := redisx.NewAutoClaimArgs(streamKey, consumer.Channel, time.Hour, "0-0", 100, consumer.Topic)
```

Comment:
Can we set this dead letter idle time via env.json?

Reply:
yes.